samples/projection-management/Program.cs

```diff
@@ -0,0 +1,312 @@
+
+
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				Console.WriteLine($"countEvents_Create already exists");
+			}
+			#endregion CreateContinuous
+
+
+		}
+
+
+
+		private static async Task Update(EventStoreProjectionManagementClient managementClient) {
+
+			#region Update
+			const string js = "fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_Update_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name, "fromAll().when()");
+			await managementClient.UpdateAsync(name, js);
+			
+			
+			try {
+				await managementClient.UpdateAsync("countEvents_Update_Not_existing", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("NotFound")) {
+				
+				Console.WriteLine("countEvents_Update_Not_existing does not exists and can not be updated");
+			}
+			#endregion Update
+		}
+
+		private static async Task ListAll(EventStoreProjectionManagementClient managementClient) {
+			#region ListAll
+			var details = managementClient.ListAllAsync();
+			await foreach (var detail in details) {
+				Console.WriteLine(
+					$@"{detail.Name}, {detail.Status}, {detail.CheckpointStatus}, {detail.Mode}, {detail.Progress}");
+			}
+			#endregion ListAll
+		}
+		
+		
+		private static async Task ListContinuous(EventStoreProjectionManagementClient managementClient) {
+			#region ListContinuous
+			var details = managementClient.ListContinuousAsync();
+			await foreach (var detail in details) {
+				Console.WriteLine(
+					$@"{detail.Name}, {detail.Status}, {detail.CheckpointStatus}, {detail.Mode}, {detail.Progress}");
+			}
+			#endregion ListContinuous
+		}
+
+		private static async Task GetStatus(EventStoreProjectionManagementClient managementClient) {
+			
+			#region GetStatus
+			const string js = "fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_status_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name,js);
+			var status = await managementClient.GetStatusAsync(name);
+			Console.WriteLine(
+				$@"{status.Name}, {status.Status}, {status.CheckpointStatus}, {status.Mode}, {status.Progress}");
+			#endregion GetStatus
+		}
+
+		private static async Task GetState(EventStoreProjectionManagementClient managementClient) {
+
+			// will have to wait for the client bu to be fixed
+			#region GetState
+			const string js =
+				"fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_State_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name,js);
+			await Task.Delay(500); //give it some time to process
+
+			var stateDocument = await managementClient.GetStateAsync(name);
+			var result  = await managementClient.GetStateAsync<Result>(name);
+			
+
+			
+			Console.WriteLine(DocToString(stateDocument));
```

Comment:
**CONCERN:** I think that this may be a bit unclear, as the user may not know what `DocToString` method is.

Reply:
it's underneath, this is not imported in the doc since there is a bug in the client , seee #142